diff --git a/config.def.h b/config.def.h
index 93a875a..de923d4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -65,5 +65,8 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ uptime, "\uf30c%s | ",   NULL },
+	{ ram_used, "RAM: %s | ",   NULL },
+	{ cpu_perc, "RAM: %s | ",   NULL },
+	{ datetime, "%s",           "%F | %T" },
 };
