diff --git a/config.def.h b/config.def.h
index 93a875a..187f60d 100644
--- a/config.def.h
+++ b/config.def.h
@@ -65,5 +65,9 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ uptime, " up %s | ",   NULL },
+	{ ram_used, "RAM: %s | ",   NULL },
+	{ cpu_perc, "CPU: %s%% | ",   NULL },
+	{ run_command, "\uf028 %s | ",   "pamixer --get-volume-human" },
+	{ datetime, "\uf073 %s",  "%F | %T " },
 };
